docs/src/main/sphinx/admin/opentelemetry.md

```diff
@@ -43,6 +43,10 @@ is accessible from the coordinator and all workers of the cluster. The preceding
 example uses a observability platform deployment available by
 HTTP at the host `observe.example.com`, port `4317`.
 
+Use the `tracing.exporter.protocol` property to configure the protocol for exporting traces. 
```

Comment:
It would be nice to use a table format to document these config properties in follow-up.